diff --git a/packages/sysutils/busybox/package.mk b/packages/sysutils/busybox/package.mk
index 7b5a8498..2f940f2e 100644
--- a/packages/sysutils/busybox/package.mk
+++ b/packages/sysutils/busybox/package.mk
@@ -7,7 +7,7 @@ PKG_NAME="busybox"
 PKG_VERSION="1.36.1"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.busybox.net"
-PKG_URL="http://busybox.net/downloads/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
+PKG_URL="https://sources.buildroot.net/${PKG_NAME}/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
 PKG_DEPENDS_HOST="gcc:host"
 PKG_DEPENDS_TARGET="toolchain busybox:host hdparm hd-idle dosfstools e2fsprogs zip unzip usbutils parted procps-ng gptfdisk libtirpc"
 PKG_DEPENDS_INIT="toolchain libc:init glibc:init libtirpc"
