diff --git a/packages/debug/edid-decode/package.mk b/packages/debug/edid-decode/package.mk
index 95a6a539..3b12281e 100644
--- a/packages/debug/edid-decode/package.mk
+++ b/packages/debug/edid-decode/package.mk
@@ -5,7 +5,7 @@ PKG_NAME="edid-decode"
 PKG_VERSION="5f723267e04deb3aa9610483514a02bcee10d9c2" # 2022-09-23
 PKG_LICENSE="None"
 PKG_SITE="https://git.linuxtv.org/edid-decode.git/"
-PKG_URL="https://repo.or.cz/edid-decode.git/snapshot/${PKG_VERSION}.tar.gz"
+PKG_URL="https://sources.buildroot.net/${PKG_NAME}/${PKG_NAME}-${PKG_VERSION}.tar.gz"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="Decode EDID data in human-readable format"
 
