diff --git a/packages/ui/emulationstation/package.mk b/packages/ui/emulationstation/package.mk
index 7b0dac762..0e561b02e 100644
--- a/packages/ui/emulationstation/package.mk
+++ b/packages/ui/emulationstation/package.mk
@@ -122,7 +122,7 @@ makeinstall_target() {
   if [ "${EMULATION_DEVICE}" = "no" ] || \
      [ "${BASE_ONLY}" = "true" ]
   then
-    cat <<EOF >${INSTALL}/etc/emulationstation/es_systems.cfg
+    cat <<EOF >${INSTALL}/usr/config/emulationstation/es_systems.cfg
 <?xml version="1.0" encoding="UTF-8"?>
 <systemList>
         <system>

