diff --git a/projects/Rockchip/packages/u-boot/package.mk b/projects/Rockchip/packages/u-boot/package.mk
index bf80822a..9f5c7a34 100644
--- a/projects/Rockchip/packages/u-boot/package.mk
+++ b/projects/Rockchip/packages/u-boot/package.mk
@@ -22,8 +22,9 @@ case ${DEVICE} in
   ;;
   RK356*)
     PKG_URL="https://github.com/u-boot/u-boot.git"
-    PKG_VERSION="e8f2404e093daf6cc3ac2b3233e3c6770d13e371"
+    PKG_VERSION="27089f1e4d11fd7e0619097b59258d0428cde2ac"
     PKG_GIT_CLONE_BRANCH="master"
+    UBOOT_IMG="u-boot.img"
   ;;
   RK3399)
     PKG_DEPENDS_TARGET+=" atf openssl:host"
