diff --git a/packages/kernel/linux/package.mk b/packages/kernel/linux/package.mk
index 3e7833cd..38282903 100644
--- a/packages/kernel/linux/package.mk
+++ b/packages/kernel/linux/package.mk
@@ -39,7 +39,7 @@ case ${DEVICE} in
     PKG_GIT_CLONE_BRANCH="main"
   ;;
   RK356*)
-    PKG_VERSION="6.8-rc6"
+    PKG_VERSION="6.8-rc3"
     PKG_URL="https://git.kernel.org/torvalds/t/${PKG_NAME}-${PKG_VERSION}.tar.gz"
   ;;
   *)
